File: javascript/resistor-color-duo/resistor-color-duo.js
export const decodedValue = (bands) => colorCode(bands[0]) * 10 + colorCode(bands[1]);

export const colorCode = (color) => COLORS.indexOf(color);

export const COLORS = [
  'black',
  'brown',
  'red',
  'orange',
  'yellow',
  'green',
  'blue',
  'violet',
  'grey',
  'white',
];

File: python/resistor-color-duo/resistor_color_duo.py
COLORS = [
        "black",
        "brown",
        "red",
        "orange",
        "yellow",
        "green",
        "blue",
        "violet",
        "grey",
        "white",
        ]

def value(colors):
    codes = [color_code(c) for c in colors[:2]]
    return codes[0] * 10 + codes[1]

def color_code(color):
    return COLORS.index(color)
